src/Microsoft.ML.CpuMath/CpuMathUtils.DotNetCoreApp.cs

```diff
@@ -0,0 +1,362 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Intrinsics.X86;
+using System;
+
+namespace Microsoft.ML.Runtime.Internal.CpuMath
+{
+    public static partial class CpuMathUtils
+    {
+        public const int CbAlign = 16;
```

Comment:
I don't think this needs to be public.  Also, `16` aligned is specific to SSE, and when we want to do AVX it would need to be changed.
Regardless, this constant can be removed if the following 2 methods are removed.